### PR TITLE
icons on android 

### PR DESCRIPTION
**Pull request by osdnk**
Fixes TEAM1-85
Figma link (if any):

## What changed (plus any additional context for devs)

I added a switch that is working correctly. Unfortunately, this is causing the restart of the app on android. I will keep working on this, but for now, let's merge this since this is an improvement anyway and I am not sure if this is possible to make it better.

I bumped the version and fix the logic on Samsung. Looks like previously they were trying ot delay the killing of the app for a bit later, but it was leading to another bug (apps god killed when opening the drawer) so I just made it simple. 

## Screen recordings / screenshots

https://user-images.githubusercontent.com/25709300/187418393-4f8322a7-38a1-4ee6-89f3-0a7a4aef13e5.mov

## What to test
Check different icons etc.

### CODE REVIEW

**Review by terrysahaidak — APPROVED** [review]
LGTM

**Review by tchayen — APPROVED** [review]
🌶️

**Review by perunt — APPROVED** [review]

File: ios/Podfile.lock [hunk 1]
```diff
@@ -1139,4 +1139,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 3d81d103721ac587f9cc40f9440d9afc96ce8bcf
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```
Comment (osdnk) [inline ios/Podfile.lock:1142]:
```suggestion
COCOAPODS: 1.11.2
```

**Review by nickbytes — APPROVED** [review]

**Review by brunobar79 — APPROVED** [review]

**Review by BrodyHughes — APPROVED** [review]
🎸

**Pull request merged**